src/components/productPage/placingOrder/PlacingOrderDetails/PlacingOrderDetails.tsx

```
@@ -13,7 +13,9 @@ function PlacingOrderDetails({ text }: Props) {
     <ol className={st.detailsList}>
       {details.map((item) => (
         <li key={uuidv4()} className={st.detailsListItem}>
-          {item}
+          <span
+            dangerouslySetInnerHTML={{ __html: item.replace(/\n/g, '<br />') }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)